src/Components/Publishing/__stories__/RelatedArticles.story.tsx

```diff
@@ -1,21 +1,17 @@
 import { storiesOf } from "@storybook/react"
 import * as React from "react"
-import { RelatedPanel } from "../Fixtures/Components"
+import { RelatedCanvas, RelatedPanel } from "../Fixtures/Components"
 import { RelatedArticlesCanvas } from "../RelatedArticles/RelatedArticlesCanvas"
 import { RelatedArticlesPanel } from "../RelatedArticles/RelatedArticlesPanel"
 
-import {
-  StandardArticle,
-} from "../Fixtures/Articles"
-
 storiesOf("Publishing/Related Articles", module)
   .add("Related Articles Panel", () => {
     return <RelatedArticlesPanel articles={RelatedPanel} />
   })
   .add("Related Articles Canvas", () => {
     return (
       <RelatedArticlesCanvas
-        articles={StandardArticle.relatedArticlesCanvas}
+        articles={RelatedCanvas}
```

Comment:
Looks like there was some leftover testing code here and in `/Fixtures/Article.ts`